src/main/java/org/troyargonauts/robot/subsystems/DriveTrain.java

```diff
@@ -1,249 +1,198 @@
      * @param turn amount we want to turn.
      * @param nerf decreases the max speed and amount we want to turn the robot.
      */
     public void cheesyDrive(double speed, double turn, double nerf) {
-        frontRight.set(((speed - turn) + Constants.DriveTrain.RIGHT_CORRECTION) * nerf);
-        frontLeft.set((speed + turn) * nerf);
+        rightSide.getMaster().set((speed + turn) * nerf);
+        leftSide.getMaster().set((speed - turn) * nerf);
     }
 
-    /** 
-     * Sets left and right motor values based on left and right parameters. 
+    /**
+     * Sets left and right motor values based on left and right parameters.
      * Robots left and right side will be controlled by different joysticks.
      * Used for mainly testing and troubleshooting.
      * @param left speed of the left side of the robot.
      * @param right speed of the right side of the robot.
      * @param nerf decreases the max speed and amount we want to turn the robot.
      */
     public void tankDrive(double left, double right, double nerf) {
-        frontRight.set((right + Constants.DriveTrain.RIGHT_CORRECTION) * nerf);
-        frontLeft.set(left * nerf);
+        rightSide.getMaster().set(right * nerf);
+        leftSide.getMaster().set(left * nerf);
     }
 
-    
-    /** 
-     * Returns encoder position based on the average value from the frontLeft and frontRight motor controller encoders.
-     * @return encoder position based on encoder values.
-     */
-    public double getPosition() {
-        return (getLeftPosition() + getRightPosition()) / 2;
-    }
 
-    /** 
-     * Returns encoder position based on the value from all the left motor controllers.
-     * @return encoder position based on frontLeft motor controller encoder.
-     */
-    public double getLeftPosition() {
-        return (frontLeftEncoderValue + middleLeftEncoderValue + backLeftEncoderValue) / 3;
-    }
-
-    /** 
-     * Returns encoder position based on the value from all the right motor controllers.
-     * @return encoder position based on frontRight motor controller encoder.
-     */
-    public double getRightPosition() {
-        return (frontRightEncoderValue + middleRightEncoderValue + backRightEncoderValue) / 3;
-    }
-
-    /**
-     * Resets the encoders to a position of 0
-     */
 
     public void resetEncoders() {
-        frontRight.getEncoder().setPosition(0);
-        middleRight.getEncoder().setPosition(0);
-        backRight.getEncoder().setPosition(0);
-        frontLeft.getEncoder().setPosition(0);
-        middleLeft.getEncoder().setPosition(0);
-        backLeft.getEncoder().setPosition(0);
+        rightSide.forEach(talonFX -> talonFX.getInternalController().getSensorCollection().setIntegratedSensorPosition(0, 50));
+        leftSide.forEach(talonFX -> talonFX.getInternalController().getSensorCollection().setIntegratedSensorPosition(0, 50));
     }
 
     /**
-     * Resets the pigeon to a yaw of 0
-     */
-    public void resetAngle() {
```

Comment:
We need to be able to reset the yaw of the gyro.

Reply:
Done